common/density/rectangular_prism.h

```diff
@@ -0,0 +1,33 @@
+/* 
+ * File:   rectangular_prism.h
+ * Author: patrick
+ *
+ * Created on February 16, 2015, 4:20 PM
+ */
+
+#ifndef RECTANGULAR_PRISM_H
+#define	RECTANGULAR_PRISM_H
+
+class RectangularPrism
```

Comment:
@ostlerc I have read the article http://www.idinews.com/quasiClass.pdf. The I neither agree with the negative viewpoint on quasiClasses because I feel that they add definition to code and in many cases are more descriptive that a tuple in their place nor is this class a quasiClass because all member variables are essentially final because there is are no setter functions.

This was a good reading excersise and I am glad of your comment. This was definitly a learning experience and I will be more mindful of it in my future coding. If I am wrong about this class feel free to respond. Perhaps post a potential solution too.